src/ledger/NetworkConfig.h

```diff
@@ -22,19 +22,19 @@ struct InitialSorobanNetworkConfig
 
     // Compute settings
     static constexpr int64_t LEDGER_MAX_INSTRUCTIONS = 1;
-    static constexpr int64_t TX_MAX_INSTRUCTIONS = 1;
+    static constexpr int64_t TX_MAX_INSTRUCTIONS = 200'000'000;
     static constexpr int64_t FEE_RATE_PER_INSTRUCTIONS_INCREMENT = 1;
-    static constexpr uint32_t MEMORY_LIMIT = 1;
+    static constexpr uint32_t MEMORY_LIMIT = 0x3200000; // 50MB
```

Comment:
How about 50 * 1024 * 1024 for readability?

Reply:
Changed.